game/world/opcode_handling/handlers/player/cheats/SpeedCheatHandler.py

```diff
@@ -12,7 +12,9 @@ def handle(world_session, socket, reader: PacketReader) -> int:
             if len(reader.data) >= 52:  # Avoid handling empty speed cheat packet.
                 speed = unpack('<f', reader.data[48:52])[0]
         else:
+            # Disconnect if the player is not a GM as I haven't found a way to change turn speed back to normal.
             Logger.anticheat(f'Player {world_session.player_mgr.player.name} ({world_session.player_mgr.guid}) tried to use speed hacks.')
+            return -1
```

Comment:
No need to disconnect here since speed cheats can be reversed (except the turn speed one).